Automattic/jetpack
# CLI: Add Jetpack Release (changelog)

Fixes #21160

#### Changes proposed in this Pull Request:

Adds the `jetpack release` command. Right now, the command only adds the first step of the process, which is generating a readme file.  

#### Jetpack product discussion



#21160

Note: This command was added only with `plugins/jetpack` in mind. It would be cool to test with other plugins/packages as well, but the plan is to dogfood the cli command for jetpack-the-plugin first before encouraging others to use it. 

We may also need to adjust some copy following the completion of the script `tools/changelogger-release.sh -b plugins/jetpack`, since creating a release branch comes after the readme.txt, which will be the next command added.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






Probably easier to test by checking out and forking so you don't gunk up the working branch with realease changes/updated changelogs/lock files, etc.
* run `jetpack changelog`
* pick a project 
* choose if it's a beta release or not
* choose the "Create Changelog.md" option.
* Make sure the process runs correctly, you see next steps, etc. 
* Also accepts passthroughs: `jetpack release -b plugins/jetpack changelog` should also work, creating changelog files for a jetpack-the-plugin beta.

Would also be helpful to read the first chunk of the fieldguide here and make sure this command looks like it covers steps 1-3 under "Generating a new changelog"
